platform/src/main/java/platform/pageobjects/Footer/FooterPage.java

```diff
@@ -9,141 +10,352 @@
 	public WebElement footerStagiaEikenLink;
 
-	@FindBy(xpath = "//a[@class='link']//font//font[contains(text(),'Information window')]")
+	@FindBy(xpath = "//a[@class='link'][contains(text(),'情報の窓口')]")
 	public WebElement informationWindowLink;
 
-	@FindBy(xpath = "//a[@class='sub-menu__text js-pitaElement']//font//font[contains(text(),'Cram school pita')]")
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'塾ピタ')]")
 	public WebElement cramSchoolPitaLink;
 
-	@FindBy(xpath = "//a[@class='sub-menu__text']//font//font[contains(text(),'Stagia University Exam')]")
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'スタギア大学受験')]")
 	public WebElement stagiaUniversityExamLink;
 
-	@FindBy(xpath = "//a[@class='sub-menu__text']//font//font[contains(text(),'Education cost consultation support')]")
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'教育費相談サポート')]")
 	public WebElement educationCostConsulatationSupportLink;
 
-	@FindBy(xpath = "//a[@class='sub-menu__text']//font//font[contains(text(),'sign up')]")
-	public WebElement signUpLink;
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'アカウントとセキュリティ')]")
+	public WebElement accountSecurity;
 
-	public void clickSignUpLinkLink() {
-		signUpLink.click();
-	}
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'会員ステータス')]")
+	public WebElement memberStatus;
 
-	public void clickInformationWindowLink() {
-		informationWindowLink.click();
-	}
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'ご利用中のサービス')]")
+	public WebElement servicesUsing;
 
-	public void clickFooterStagiaEikenLink() {
+	@FindBy(xpath = "//a[@class='sub-menu__text'][contains(text(),'ご注文履歴')]")
+	public WebElement orderHistory;
 
-		footerStagiaEikenLink.click();
+	@FindBy(xpath = "//a[contains(text(),'退会')]")
+	public WebElement withdraw;
 
-	}
+	@FindBy(xpath = "//a[contains(text(),'ログアウト')]")
+	public WebElement logout;
+
+	@FindBy(xpath = "//a[contains(text(),'よくある質問・お問合せ')]")
+	public WebElement accountQnA;
+
+	@FindBy(xpath = "//a[contains(text(),'利用規約・個人情報の取扱')]")
+	public WebElement accountTerms;
 
-	public void clickFooterLearningWindowLink() {
+	@FindBy(xpath = "//a[contains(text(),'個人情報保護方針')]")
+	public WebElement accountPrivacy;
 
-		footerLearningWindowLink.click();
+	@FindBy(xpath = "//a[contains(text(),'特定商取引法に基づく表示')]")
+	public WebElement accountTax;
 
+	public void switchToPreviousTab() {
+		ArrayList<String> tab = new ArrayList<>(driver.getWindowHandles());
+		driver.switchTo().window(tab.get(1));
+		driver.close();
+		driver.switchTo().window(tab.get(0));
+		// driver.close();
 	}
 
-	public void clickFooterCasecLink() {
+	public void footerFunctions() throws Exception {
 
-		footerCasecLink.click();
+		CommonFunctions.scrolltoElement(termsOfUseLink);
+		if (CommonFunctions.isElementClickable(termsOfUseLink)) {
+			termsOfUseLink.click();
+			log.info("Terms of Use Link is working");
+			Thread.sleep(3000);
```

Comment:
Remove Thread.sleep mention from code.
Use wait from CommonFunctions for the same

Reply:
Removed Thread.sleep function